src/azure-cli/azure/cli/command_modules/vm/_image_builder.py

```diff
@@ -423,7 +429,8 @@ def create_image_template(  # pylint: disable=too-many-locals, too-many-branches
     if source_dict['type'] == _SourceType.PLATFORM_IMAGE:
         template_source = ImageTemplatePlatformImageSource(**source_dict)
     elif source_dict['type'] == _SourceType.ISO_URI:
-        template_source = ImageTemplateIsoSource(**source_dict)
+        raise CLIError('usage error: Source type ISO URI is not supported.')
```

Comment:
>usage [](start = 24, length = 5)

Usage error: source type ISO URI is not supported.

Reply:
I was supported but not supported now. The service is evolving.